scripts/pi-hole/php/loginpage.php

```diff
@@ -30,34 +30,36 @@
           <input type="password" id="loginpw" name="pw" class="form-control" placeholder="Password" autofocus>
           <span class="fa fa-key form-control-feedback"></span>
         </div>
+        <div class="row">
+          <div class="col-xs-12 col-md-12">
+            <button type="submit" class="btn btn-primary form-control"><i class="fas fa-sign-in-alt"></i>&nbsp;&nbsp;&nbsp;Log in</button>
+          </div>
+        </div>
+        <br>
         <div class="row">
           <div class="col-xs-8 hidden-xs hidden-sm">
-          <ul>
-            <li><kbd>Return</kbd> &rarr; Log in and go to requested page (<?php echo $scriptname; ?>)</li>
-            <li><kbd>Ctrl</kbd>+<kbd>Return</kbd> &rarr; Log in and go to Settings page</li>
-          </ul>
+            <ul>
+              <li><kbd>Return</kbd> &rarr; Log in and go to requested page (<?php echo $scriptname; ?>)</li>
+              <li><kbd>Ctrl</kbd>+<kbd>Return</kbd> &rarr; Log in and go to Settings page</li>
+            </ul>
           </div>
           <div class="col-xs-12 col-md-4">
-              <div class="pull-left">
-                <div>
-                  <input type="checkbox" id="logincookie" name="persistentlogin">
-                  <label for="logincookie">Remember me for 7 days</label>
-                </div>
-            </div>
-            <button type="submit" class="btn btn-primary pull-right"><i class="fas fa-sign-in-alt"></i>&nbsp;&nbsp;&nbsp;Log in</button>
+              <div>
+                <input type="checkbox" id="logincookie" name="persistentlogin">
+                <label for="logincookie">Remember me for 7 days</label>
+              </div>
           </div>
         </div>
         <br>
         <div class="row">
           <div class="col-xs-12">
             <div class="box box-<?php if (!$wrongpassword) { ?>info<?php } else { ?>danger<?php }
-            if (!$wrongpassword) { ?> collapsed-box<?php } ?> box-solid">
+                                                                                        if (!$wrongpassword) { ?> collapsed-box<?php } ?>">
```

Comment:
Many new spaces?

Also, this should be combined with the `if()` directly before this one.

Reply:
I am not sure how I managed the many extra spaces!!